docs/business-apps/power-automate/sharepoint-connector-actions-triggers.md

```diff
@@ -4,237 +4,205 @@ ms.date: 03/13/2020
 ---
 
 # Microsoft SharePoint Connector in Power Automate
-
 Microsoft SharePoint Connector in Power Automate supports the following flow triggers and actions.
 
 ## SharePoint triggers
+SharePoint triggers allows you to create flows that monitor for changes in a SharePoint list or library. If one or more changes occur in that subscribed list, then that flow will be triggered to run. 
```

Comment:
SharePoint triggers let you create flows that monitor for changes in a SharePoint list or library. If one or more changes occur in a subscribed list, that flow is triggered to run.